README.md

```diff
@@ -1 +1,213 @@
-# JavaScript-capstone-project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <!-- <img src="murple_logo.png" alt="logo" width="140"  height="auto" /> -->
+  <br/>
+
+  <!-- <h3><b>Microverse README Template</b></h3> -->
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 [tv-movies] <a name="about-project"></a>](#-tv-movies-)
+  - [🛠 Built With <a name="HTML and CSS"></a>](#-built-with-)
+    - [Tech Stack <a name="Front end"></a>](#tech-stack-)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo <a name="https://github.com/waelelsafty07/JavaScript-capstone-project/dist"></a>](#-live-demo-)
+  - [💻 Getting Started <a name="getting-started"></a>](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Author <a name="authors">Wael Elsafty</a>](#-author-wael-elsafty)
+  - [🔭 Future Features <a name="future-features"></a>](#-future-features-)
+  - [🤝 Contributing <a name="contributing"></a>](#-contributing-)
+  - [⭐️ Show your support <a name="support"></a>](#️-show-your-support-)
+  - [🙏 Acknowledgments <a name="acknowledgements"></a>](#-acknowledgments-)
+  - [📝 License <a name="license"></a>](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [tv-movies] <a name="about-project"></a>
+
+> In this project, Tv movies webiste displays movies submitted by different user. It also allows you to submit your revision. All data is preserved thanks to the external tvmaze API service.
+
+**[tv-movies]** is a website that is used test the knowledge of hit api and how work on group in same project
+
+## 🛠 Built With <a name="HTML and CSS"></a>
+
+### Tech Stack <a name="Front end"></a>
+
+> This is a desk-top browser website.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">HTML</a></li>
+    <li><a href="#">CSS</a></li>
+    <li><a href="#">JS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features
+
+- **[intialize Project.]**
+
```

Comment:
I think it would be a good idea to leave a more descriptive feature such as "Initialized project setup with Webpack and Jest library"